setup.json

```diff
@@ -70,11 +70,11 @@
         "docs": [
             "docutils==0.15.2",
             "pygments~=2.5",
-            "pydata-sphinx-theme~=0.3.0",
-            "sphinx~=2.2",
+            "pydata-sphinx-theme~=0.4.0",
+            "sphinx~=3.2",
             "sphinxcontrib-details-directive~=0.1.0",
-            "sphinx-panels~=0.4.0",
-            "sphinx-copybutton~=0.2.12",
+            "sphinx-panels~=0.5.0",
+            "sphinx-copybutton~=0.3.0",
```

Comment:
Is there a reason that we restrict the sphinx extensions to a minor version?

Reply:
I think probably just because they haven't release a v1.0 yet. So far we have been maintaining the rule of thumb that for packages with at least a v1 we pin to the major version, whereas for v0.* releases we pin to the minor version. This is because typically packages are a bit more liberal with breaking the interface before a v1 release.

Reply:
Indeed, these packages are both from EBP and that version convention is specifically outlined (by me 😄) https://github.com/executablebooks/.github/blob/master/CONTRIBUTING.md#releases-and-change-logs

Reply:
Makes sense. 👍